commons/commons-serialization/src/main/kotlin/ac/uk/ebi/biostd/SerializationService.kt

```diff
@@ -23,6 +23,18 @@ class SerializationService(
 
     fun serializeSubmission(submission: Submission, format: SubFormat) = serializeElement(submission, format)
 
+    inline fun <reified T> deserializeElement(
+        element: String,
+        format: SubFormat
+    ) = deserializeElement(element, format, T::class.java) as T
+
+    fun <T> deserializeElement(element: String, format: SubFormat, type: Class<out T>? = null) = when (format) {
+        XML -> xmlSerializer.deserialize(element)
+        JSON -> jsonSerializer.deserialize(element)
+        JSON_PRETTY -> jsonSerializer.deserialize(element)
+        TSV -> tsvSerializer.deserializeElement(element, type!!)
```

Comment:
We discussed this change, this is required because none of the other deserializers require to specify the class that is being deserialized so, the `type` argument isn't necessary for them but it shouldn't be null for the TSV deserializer.